index.html

```diff
@@ -2021,6 +2021,13 @@ <h3>Securing Mechanisms</h3>
 algorithm</a> section.
           </li>
           <li>
+MUST document post-processing considerations for data returned from verification
+processes. This includes, but is not limited to, what information can and cannot
+be used for decision making, for what period of time after verification, when
+additional information is required, and when care is necessary even when the
+returned data was properly secured.
```

Comment:
I don't understand how a securing method could dictate, in any meaningful way, what could be used for decision making or not, so I've reworded this to focus on the securing mechanism describing the nature of the data being returned instead:

```suggestion
processes. This includes, but is not limited to, what information has been
secured, for what period of time after verification, when or what additional
information might be required to assist in decision making, and when
care is necessary even when the returned data was properly secured.
```